packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1841,10 +1841,13 @@ export default class Meeting extends StatelessWebexPlugin {
 
     const LOG_HEADER = 'Meeting:index#muteAudio -->';
 
+    // First, stop sending the local audio media
     return logRequest(this.audio.toggle({
       mute: true,
       self: true
     })
+      // Next, send the control to mute the participant on the server
+      .then(() => this.mute(this.members.selfId, true))
```

Comment:
I tried to do this in the toggle code but it would require pulling in all of the participant request data and this made things cleaner.